Improve auto-translation of editor's top & bottom bars

timothyqiu opened the pull request:
Part of https://github.com/godotengine/godot/issues/104574

Cleaned up auto translation for:

- The main menu
- The run bar
- The rendering method selector tooltip
- The update spinner
- The bottom panel buttons

timothyqiu committed:
Improve auto-translation of editor's top & bottom bars

- The main menu
- The run bar
- The update spinner
- The bottom panel buttons

KoBeWi commented on editor/editor_node.cpp:
```diff
@@ -7035,20 +7035,23 @@ void EditorNode::_renderer_selected(int p_which) {
 	if (p_mark_overridden) {
-		item_text += " " + TTR("(Overridden)");
+		// TRANSLATORS: The placeholder is the rendering method that has overridden the default one.
+		renderer->add_item(vformat("%s (Overridden)", rendering_method));
```
```suggestion
		renderer->add_item(vformat(TTR("%s (Overridden)"), rendering_method));
```

KoBeWi commented:
I don't see any code that would update renderer names after language change. Is it oversight or you plan to add it later? (it doesn't seem easy to do)

timothyqiu commented:
> I don't see any code that would update renderer names after language change. Is it oversight or you plan to add it later? (it doesn't seem easy to do)

Ah yes, I'll make a separate PR for the rendering methods dropdown.

Updated. Removed those rendering method selector changes that are meaningless from the perspective of this PR.

Repiteo commented:
Thanks!
